### ISSUE TITLE
Error while trying to run test

### ISSUE THREAD

**Issue by max-monastirsky** [NVlabs/VoxFormer#42]
Hello, I followed the installation guide and tried to run test, and got this error:

```python
Traceback (most recent call last):
  File "./tools/test.py", line 19, in <module>
    from mmdet3d.apis import single_gpu_test
  File "/home/max/Projects/vox/mmdetection3d/mmdet3d/apis/__init__.py", line 2, in <module>
    from .inference import (convert_SyncBN, inference_detector,
  File "/home/max/Projects/vox/mmdetection3d/mmdet3d/apis/inference.py", line 11, in <module>
    from mmdet3d.core import (Box3DMode, CameraInstance3DBoxes,
  File "/home/max/Projects/vox/mmdetection3d/mmdet3d/core/__init__.py", line 2, in <module>
    from .anchor import *  # noqa: F401, F403
  File "/home/max/Projects/vox/mmdetection3d/mmdet3d/core/anchor/__init__.py", line 2, in <module>
    from mmdet.core.anchor import build_prior_generator
  File "/home/max/miniconda3/envs/om/lib/python3.8/site-packages/mmdet/core/__init__.py", line 2, in <module>
    from .bbox import *  # noqa: F401, F403
  File "/home/max/miniconda3/envs/om/lib/python3.8/site-packages/mmdet/core/bbox/__init__.py", line 7, in <module>
    from .samplers import (BaseSampler, CombinedSampler,
  File "/home/max/miniconda3/envs/om/lib/python3.8/site-packages/mmdet/core/bbox/samplers/__init__.py", line 9, in <module>
    from .score_hlr_sampler import ScoreHLRSampler
  File "/home/max/miniconda3/envs/om/lib/python3.8/site-packages/mmdet/core/bbox/samplers/score_hlr_sampler.py", line 2, in <module>
    from mmcv.ops import nms_match
  File "/home/max/miniconda3/envs/om/lib/python3.8/site-packages/mmcv/ops/__init__.py", line 2, in <module>
    from .assign_score_withk import assign_score_withk
  File "/home/max/miniconda3/envs/om/lib/python3.8/site-packages/mmcv/ops/assign_score_withk.py", line 5, in <module>
    ext_module = ext_loader.load_ext(
  File "/home/max/miniconda3/envs/om/lib/python3.8/site-packages/mmcv/utils/ext_loader.py", line 13, in load_ext
    ext = importlib.import_module('mmcv.' + name)
  File "/home/max/miniconda3/envs/om/lib/python3.8/importlib/__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
ModuleNotFoundError: No module named 'mmcv._ext'

```

I can't find a way to fix this,
Thanks

**Comment by max-monastirsky** [comment]
Managed to fix the previous error by uninstalling mmcv-full and reinstalling with:

`pip install mmcv-full==1.4.0 --no-cache --no-binary :all:`

But now I have encountered the following error:
```python
Traceback (most recent call last):
  File "./tools/test.py", line 19, in <module>
    from mmdet3d.apis import single_gpu_test
  File "/home/max/Projects/vox/mmdetection3d/mmdet3d/apis/__init__.py", line 2, in <module>
    from .inference import (convert_SyncBN, inference_detector,
  File "/home/max/Projects/vox/mmdetection3d/mmdet3d/apis/inference.py", line 11, in <module>
    from mmdet3d.core import (Box3DMode, CameraInstance3DBoxes,
  File "/home/max/Projects/vox/mmdetection3d/mmdet3d/core/__init__.py", line 3, in <module>
    from .bbox import *  # noqa: F401, F403
  File "/home/max/Projects/vox/mmdetection3d/mmdet3d/core/bbox/__init__.py", line 5, in <module>
    from .iou_calculators import (AxisAlignedBboxOverlaps3D, BboxOverlaps3D,
  File "/home/max/Projects/vox/mmdetection3d/mmdet3d/core/bbox/iou_calculators/__init__.py", line 2, in <module>
    from .iou3d_calculator import (AxisAlignedBboxOverlaps3D, BboxOverlaps3D,
  File "/home/max/Projects/vox/mmdetection3d/mmdet3d/core/bbox/iou_calculators/iou3d_calculator.py", line 6, in <module>
    from ..structures import get_box_type
  File "/home/max/Projects/vox/mmdetection3d/mmdet3d/core/bbox/structures/__init__.py", line 2, in <module>
    from .base_box3d import BaseInstance3DBoxes
  File "/home/max/Projects/vox/mmdetection3d/mmdet3d/core/bbox/structures/base_box3d.py", line 6, in <module>
    from mmdet3d.ops.iou3d import iou3d_cuda
  File "/home/max/Projects/vox/mmdetection3d/mmdet3d/ops/__init__.py", line 20, in <module>
    from .roiaware_pool3d import (RoIAwarePool3d, points_in_boxes_batch,
  File "/home/max/Projects/vox/mmdetection3d/mmdet3d/ops/roiaware_pool3d/__init__.py", line 1, in <module>
    from .points_in_boxes import (points_in_boxes_batch, points_in_boxes_cpu,
  File "/home/max/Projects/vox/mmdetection3d/mmdet3d/ops/roiaware_pool3d/points_in_boxes.py", line 3, in <module>
    from . import roiaware_pool3d_ext
ImportError: /home/max/Projects/vox/mmdetection3d/mmdet3d/ops/roiaware_pool3d/roiaware_pool3d_ext.cpython-38-x86_64-linux-gnu.so: undefined symbol: _ZN3c1015SmallVectorBaseIjE8grow_podEPvmm
```

**Comment by max-monastirsky** [comment]
If any one is wondering, managed to fix this by downgrading my cudatools to 11.1, make sure to check your cuda runtime API version `nvcc -V`.
Then installing torch 1.9+cu11.1, deleting and reinstalling mmcv-full using `pip install mmcv-full==1.4.0 --no-cache --no-binary :all:`. And finally recompiling mmdet3d.
